ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/FooCommandGenerator.scala

```diff
@@ -114,6 +120,9 @@ final class FooCommandGenerator(
           case "Foo3" => Foo3(signatory, observers, payload, keyId = fooKeyId).create.command
         }
     }
+    if (config.allowNonTransientContracts) {
```

Comment:
Does the maintenance of the contract keys needs to be guarded ?

Reply:
It doesn't in that it shouldn't break anything but on the other hand it serves to purpose when non-transient contracts flag is off